src/scripts/main.js

```diff
@@ -1 +1,51 @@
 'use strict';
+
+const clicks = ['click', 'contextmenu'];
+
+const firstPromise = new Promise((resolve, reject) => {
+  setTimeout(() => reject(new Error(`First promise was rejected`)), 3000);
+
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
```

Comment:
The `firstPromise` resolves on any click event. Ensure it only resolves on a left click as specified in the task description.